runner/llamarunner/runner.go

```diff
@@ -130,6 +130,8 @@ func (s *Server) NewSequence(prompt string, images []llm.ImageData, params NewSe
 			return nil, errorInputTooLong
 		}
 
+		// Embeddings and generation both return the same too-long error when truncate is enabled
```

Comment:
I might just remove this comment - it seems mostly relevant in the context of this PR but I don't know that it is really helpful going forward.